molecule/default/tests/test_default.py

```diff
@@ -1,25 +1,130 @@
 import os
 import re
-
 import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
-@pytest.mark.skip(reason='TODO: will update the test to run '
-                         'cinder command on controller')
+@pytest.mark.jira('asc-222')
+@pytest.mark.skip(reason='this test can not run on storage host')
 def test_cinder_service(host):
+    """
+    TODO: will update the test to run cinder command on controller node
+    TODO: since the test cannot run on storage hosts.
+    """
     cmd = "sudo bash -c \"source /root/openrc; cinder service-list\""
     output = host.run(cmd)
     assert ("cinder-volume" in output.stdout)
 
 
+@pytest.mark.jira('asc-222')
 def test_cinder_lvm_volume(host):
+    """Test 2a: Check the Cinder Nodes: volume group
+    Args:
+        host(string): A hostname in dynamic_inventory.json/molecule.yml
+    Returns:
+        int: Exit_codes (test failed if exit_code is not zero)
+    """
+
     output = host.run('vgs cinder-volumes')
     assert re.search("cinder-volumes\s+[0-9]*\s+[0-9]*\s+", output.stdout)
 
 
+@pytest.mark.jira('asc-222')
 def test_cinder_volume_group(host):
+    """Test 2a: Check the Cinder config file
+    Args:
+        host(string): A hostname in dynamic_inventory.json/molecule.yml
+    Returns:
+        int: Exit_codes (test failed if exit_code is not zero)
```

Comment:
py.test function have no return value so you should just leave this out of the docstring.